.github/workflows/publish-aws-lambda-otel-extension.yml

```diff
@@ -112,3 +116,10 @@ jobs:
           npm publish
           cd ../aws-lambda-otel-extension-dist
           npm publish
+
+      - name: Publish release notes
+        run: |
+          cd packages/aws-lambda-otel-extension
+          TEMP_ARRAY=($(echo $GITHUB_REF | tr "/" "\n"))
+          TAG=${TEMP_ARRAY[@]: -1}
+          npx github-release-from-cc-changelog $TAG
```

Comment:
It looks like this is a package you created 😎 Seems pretty cool. Do we use this in any of our other open source projects?

Reply:
Yes, it's used in the Framework and all related packages